### PR TITLE
Allow deploying the ansible_tower collector

### PR DESCRIPTION
**Pull request by bdunne**

### CODE REVIEW

File: lib/topological_inventory/orchestrator/worker.rb [hunk 1]
```diff
@@ -162,10 +162,13 @@ def authentication_with_password(id, tenant_account)
       def collector_definitions
         @collector_definitions ||= begin
           {
-            "amazon"    => {
+            "amazon"        => {
               "image" => "topological-inventory-amazon:#{collector_image_tag}"
             },
-            "openshift" => {
+            "ansible_tower" => {
```
Comment (carbonin) [inline lib/topological_inventory/orchestrator/worker.rb]: This needs to match https://github.com/ManageIQ/sources-api/blob/master/db/seeds.rb#L46 I think

**Comment by carbonin** [comment]
I think this will have to wait for https://github.com/RedHatInsights/e2e-deploy/pull/459

**Comment by bdunne** [comment]
https://github.com/RedHatInsights/e2e-deploy/pull/459 is merged and I see images in buildfactory

**Comment by bdunne** [comment]
cc @Fryguy

**Pull request merged**